src/table/boot.rs

```diff
@@ -14,8 +14,8 @@ pub struct BootServices {
     header: Header,
 
     // Task Priority services
-    raise_tpl: extern "win64" fn(new_tpl: Tpl) -> Tpl,
-    restore_tpl: extern "win64" fn(old_tpl: Tpl),
+    raise_tpl: unsafe extern "win64" fn(new_tpl: Tpl) -> Tpl,
+    restore_tpl: unsafe extern "win64" fn(old_tpl: Tpl),
```

Comment:
EFI entry points are marked `unsafe` if they accept some inputs that can trigger UB (which, surprisingly, is the case for Raise/RestoreTpl).